code_folder/F1_plotting.py

```diff
@@ -106,7 +106,7 @@ def capacities(user_input, project_data, assets, capacities):
         dfcapacities.plot.bar(
             x="items",
             y="capacities",
-            title="Optimal additional capacities: "
+            title="Optimal additional capacities (kW): "
```

Comment:
(kW) is not always the unit, can you change this to (kW/kWh/kWhp)?

Reply:
done!